cosmos-auth/src/server.js

```diff
@@ -0,0 +1,90 @@
+/**
+ * Copyright 2015 Telefonica Investigación y Desarrollo, S.A.U
+ *
+ * This file is part of fiware-cosmos (FI-WARE project).
+ *
+ * fiware-cosmos is free software: you can redistribute it and/or modify it under the terms of the GNU Affero
+ * General Public License as published by the Free Software Foundation, either version 3 of the License, or (at your
+ * option) any later version.
+ * fiware-tidoop is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the
+ * implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero General Public License
+ * for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License along with fiware-cosmos. If not, see
+ * http://www.gnu.org/licenses/.
+ *
+ * For those usages not covered by the GNU Affero General Public License please contact with
+ * francisco dot romerobueno at telefonica dot com
+ */
+
+/**
+ * Http REST server for cosmos auth token generator
+ *
+ * Author: frb
+ */
+
+// Module dependencies
+var Hapi = require('hapi');
+var boom = require('boom');
+var packageJson = require('../package.json');
+var config = require('../conf/cosmos-auth.json');
+var client = require('./client.js');
+
+// Create a Hapi server with a host and port
+var server = new Hapi.Server();
+
+server.connection({
+    host: 'localhost',
```

Comment:
I guess you plan to make this host configurable in the future, right? Leaving it as localhost won't make it visible "from the outside world", right? :)